ThunderCloud/QuizSliderViewController.swift

```diff
@@ -36,6 +122,7 @@ class QuizSliderViewController: UIViewController, QuizQuestionViewController {
         if let image = question.sliderImage {
             
             imageView.accessibilityLabel = question.image?.accessibilityLabel
+            imageView.isAccessibilityElement = question.image?.accessibilityLabel != nil
```

Comment:
Within `ImageView`, could `isAccessibilityElement` be determined based on whether the accessibility label has been set? Might mean that all of these `!= nil` checks could be removed? :)

Reply:
I would like to avoid this, as `ImageView` is provided by `ThunderTable` and therefore don't think we should force this logic upon users of that framework :) Especially as it's much more likely to be used by other devs!

Reply:
It's a great shout though... may in future subclass `ImageView` in ThunderCloud to do this, but pretty pressed for time atm :D

Reply:
That's a good shout actually, if it's provided by another framework then best leave it be for a potential future improvement :D